transports/transports-deciding.md

```diff
@@ -0,0 +1,196 @@
+- Supports both queues and business data in a single backup, making it much easier to restore your system into a consistent state.
+
+#### Disadvantages
+
+- Does not offer a native publish-subscribe mechanism, therefore it requires NServiceBus persistence to be configured for storing event subscriptions. [Explicit routing for publish/subscribe](/nservicebus/messaging/routing.md#event-routing-message-driven) must also be specified.
+- Maximum throughput is for the entire system, instead of a single endpoint.
+- The transport polls the queue table even if there are no messages to process.
+- Depending on throughput it adds additional load on your SQL Server installation.
+
+#### When would you select this transport
+
+- If you cannot introduce another queuing technology
+- If you have invested in a SQL Server cluster
+- If you don’t want to invest in possible training for another queuing technology.
+- If you have a legacy application and need SQL triggers for integration with NServiceBus.
+
+
+
+## RabbitMQ Transport
+
+RabbitMQ is a popular message broker that is used on many development platforms. It can be used both on-premise and in the cloud.
+
+#### Advantages
+
+- There is a large community around RabbitMQ
+- Provides native reliability and high-availability features.
+- Offers native publish-subscribe features, therefore doesn’t need NServiceBus persistence.
+- Allows for integration with applications written in other languages, using native RabbitMQ features through a wide range of [supported clients](<https://www.rabbitmq.com/devtools.html>).
+- Supports [competing consumer pattern](http://www.enterpriseintegrationpatterns.com/patterns/messaging/CompetingConsumers.html) out of the box. Messages are received by instances in a round-robin fashion without additional configuration.
+
+#### Disadvantages
+
+- Running RabbitMQ in a cluster, which is highly recommended, requires deeper operational knowledge of RabbitMQ. Not all companies have the same level of expertise as with other technologies, like SQL Server. This may require additional training.
+- Doesn’t handle [network partitions](https://www.rabbitmq.com/partitions.html) well; partitioning across a WAN requires using dedicated features
+- Requires careful consideration for duplicate messages, e.g. using the outbox feature or making all endpoints idempotent.
+- Might require covering additional costs of [commercial RabbitMQ license and support](<https://www.rabbitmq.com/services.html>).
+
+#### When would you select this transport
+
+- If you want native integration with other development platforms
+- If you already are running RabbitMQ and don’t want another queuing technology
+
+### Amazon SQS
+
+This is a popular transport if your system is hosted inside AWS, Amazon its cloud offering.
+
+#### Advantages
+
+- Fully managed turn-key infrastructure
```

Comment:
Does this also apply to the Azure transports?

Reply:
Yes.

Reply:
I've added it to the Azure transports in https://github.com/Particular/docs.particular.net/pull/4112